catalyst/contrib/data/cv/reader.py

```diff
@@ -52,7 +52,7 @@ class MaskReader(ReaderSpec):
     def __init__(
         self,
         input_key: str,
-        output_key: str,
+        output_key: Optional[str],
```

Comment:
`Optional`?

Reply:
it is better to use `Optional` if default value is `None` (as proposed in [mypy](https://github.com/python/mypy))